modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/parser/SchedulerParser.java

```diff
@@ -46,16 +42,6 @@ private SchedulerParser() {
      */
     public static Scheduler parse(ScheduledExecutorService scheduledExecutorService, Schedulable
```

Comment:
Do we need this class ?

Reply:
We do not need this class. I will remove it.

Reply:
kept the class to provide a wrapper to add the scheduler to a list in siddhiAppContext